tst/EnergyPlus/unit/FanCoilUnits.unit.cc

```diff
@@ -657,41 +655,39 @@ TEST_F(EnergyPlusFixture, MultiStage4PipeFanCoilCoolingTest)
     state->dataWaterCoils->WaterCoil(1).MaxWaterMassFlowRate = HotWaterMassFlowRate;
 
     for (int l = 1; l <= state->dataPlnt->TotNumLoops; ++l) {
-        auto &loop(state->dataPlnt->PlantLoop(l));
```

Comment:
A lot of these lines started to show up as unused code warnings after the `.allocate() `calls were removed. This happened because `loop` was only used to call the `.allocate()` method.  Cleaned up all of these.

Reply:
The best code is deleted code. Not even kidding.